### PR TITLE
Feature/event logging

### PR DESCRIPTION
**Pull request by mtrupkin**

### CODE REVIEW

File: student/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 		<testscoring.version>0.0.1-SNAPSHOT</testscoring.version>
 		<progman-client.version>3.0.0</progman-client.version>
 		<sb11-mna-client.version>0.0.4-SNAPSHOT</sb11-mna-client.version>
-		<sb11-shared-code.version>3.0.0</sb11-shared-code.version>
+		<sb11-shared-code.version>3.0.1-SNAPSHOT</sb11-shared-code.version>
```
Comment (wtritch) [inline student/pom.xml:29]: Please release sb11-shared-code and bump to a released version before merging. =)

File: student/src/main/java/tds/student/web/handlers/TestShellHandler.java [hunk 2]
```diff
@@ -183,9 +186,10 @@ private String parseOutLatencies (Map<String, String> formParams) {
   public ResponseData<String> PauseTest (@RequestParam Map<String, String> formParams, @RequestParam (value = "reason", required = false) String reason, HttpServletRequest request)
       throws TDSSecurityException, ReturnStatusException {
     // check if authenticated
+    EventLogger.info(StudentEventLogger.APP, StudentEventLogger.StudentLogEvent.PAUSE_EXAM, Checkpoint.ENTER.name(),
```
Comment (wtritch) [inline student/src/main/java/tds/student/web/handlers/TestShellHandler.java:189]: Should this be StudentEventLogger.info ?

File: student/src/main/java/tds/student/web/handlers/TestShellHandler.java [hunk 3]
```diff
@@ -183,9 +186,10 @@ private String parseOutLatencies (Map<String, String> formParams) {
...
+      request.getSession().getId(), null, null);
```
Comment (wtritch) [inline student/src/main/java/tds/student/web/handlers/TestShellHandler.java:190]: maybe pull request.getSession().getId() into a sessionId variable?

**Review by wtritch — CHANGES_REQUESTED** [review]
Just requesting a release of sb11-shared-code.

**Comment by mtrupkin** [comment]
Researching alternate approach to event logging in the feature/event-interceptor branch.
Looking to inspect the http request/response in order to determine the event type and event specific data.

**Pull request closed without merging**